sources/packages/backend/apps/db-migrations/src/sql/ApplicationOfferingChangeRequests/Create-application-offering-change-requests.sql

```diff
@@ -0,0 +1,52 @@
+CREATE TABLE IF NOT EXISTS sims.application_offering_change_requests (
```

Comment:
As I understood from a previous PR, we won't add `IF EXISTS`/`IF NOT EXISTS` anymore.

Reply:
I had it in my mind, but then I was confused about whether our discussion was not to use `IF EXISTS/IF NOT EXISTS` for ALTER tables or in all `.sql` comments, I actually tried to find the old chat in dev, and I couldn't find it. 
so I thought, I will wait for the dev comment. 
 Thanks for the comments. I will address it

Reply:
The idea is that using IF EXISTS/IF NOT EXISTS can hide some issues in the migrations. The migrations should enforce whether the item exists or not because the DB is supposed to be changed only through a migration.
If a migration is executed to remove an item and the item is not present, either the migration is wrong or the DB it is in a state where the migration should not be executed.